### ISSUE TITLE
Building on ARM

### ISSUE THREAD

**Issue by CentaurWarchief** [graphql-go/graphql#95]
I can't build this library for ARM:
```
../../graphql-go/graphql/scalars.go:11: constant 9007199254740991 overflows int
../../graphql-go/graphql/scalars.go:12: constant -9007199254740991 overflows int
```

```
GOOS=linux GOARCH=arm
```

go1.5.1

```
➜ ~ % uname -a
Linux raspberrypi 4.1.13-v7+ #826 SMP PREEMPT Fri Nov 13 20:19:03 GMT 2015 armv7l GNU/Linux
```

**Comment by CentaurWarchief** [comment]
Apparently, I downloaded the package again and it worked.

**Comment by chris-ramon** [comment]
Hi @CentaurWarchief thanks for letting us know about it - just for the record, this was fixed on #83 and the discussion [here](https://github.com/graphql-go/graphql/issues/24).

**Comment by CentaurWarchief** [comment]
Hey @chris-ramon, thanks. It's working 100%, I think I can cut the `Apparently` :stuck_out_tongue_winking_eye:
